packages/valory/skills/transaction_settlement_abci/behaviours.py

```diff
@@ -201,6 +201,7 @@ def _get_tx_data(
         tx_data["tx_digest"] = cast(str, tx_digest)
 
         nonce = Nonce(int(cast(str, message.raw_transaction.body["nonce"])))
+        fallback_gas = message.raw_transaction.body["gas"]
```

Comment:
For the repricing, we need a gas estimate, otherwise the transaction will fail!

Reply:
On second thought, this makes sense. Reprice gas estimation **must** always be manual. 

- The first instance why it might fail if a fallback is not provided is because the eth node doesnt support the using the last state/block.

- Secondly, even if the estimation during repricing goes through with the last state, we cannot expect for it to result in a different gas from what the original tx was submitted with, given that the original tx is estimated automatically.